templates/search/blocks/search_filters.html

```diff
@@ -48,29 +48,23 @@ <h3 class="sr-only">Edit filters</h3>
                             <h4 class="search-filters__accordion-section-heading" id="record_covering_date">Dates</h4>
                         </legend>
 
+                        <div>{{ form.covering_date_from.label_tag }}</div>
```

Comment:
I don't think `label_tag` is right here. What we really should have here is a `fieldset` tag wrapping the entire field, and the label value used to set the `legend` for that fieldset.

Reply:
Hi @JohnHeeryTNA , does this need to change before the PR can be reviewed? Thanks.

Reply:
@sdwilkes I've had a look at this and I think the rendered html is correct and following the GDS pattern: https://design-system.service.gov.uk/components/date-input/

According to that we should only have one <fieldset> which wraps the input fields, which is what we have. 

@ahosgood do you have a take on this?

Reply:
I can't seem to see this in the list of changes (https://github.com/nationalarchives/ds-wagtail/pull/1074/files) but yes, there should be a single `<fieldset>` that encompasses all three date inputs as a set of fields and each `<input>` (day, month and year) should have a corresponding `<label>` element which describes it.

Reply:
...and a single `<legend>` for the `<fieldset>` which describes the set of fields as a whole